taiga_halo2/src/circuit/vp_examples/field_addition.rs

```diff
@@ -69,36 +78,59 @@ impl FieldAdditionValidityPredicateCircuit {
             b,
         }
     }
+}
+
+impl ValidityPredicateInfo for FieldAdditionValidityPredicateCircuit {
+    fn get_spend_notes(&self) -> &[Note; NUM_NOTE] {
```

Comment:
Can we have these methods `get_spend_notes` and `get_output_notes` with a default implementation? It may not be immediate but probably there is a way around